test/api/entry.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { BaseEntry } from 'src';
+import { Entry } from '../../src/lib/entry';
+import { stackInstance } from '../utils/stack-instance';
+import { TEntry } from './types';
+
+const stack = stackInstance();
+
+describe('Entry API tests', () => {
+  it('should check for entry is defined', async () => {
+    const result = await makeEntry('blt09f7d2d46afe6dc6').fetch<TEntry>();
```

Comment:
remove the credentials from the file @nadeem-cs

Reply:
Resolved in [this](https://github.com/contentstack/contentstack-javascript/pull/110) PR.